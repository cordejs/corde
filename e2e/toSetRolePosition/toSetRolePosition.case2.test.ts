import Utils from "../testUtils";
import { runCLI } from "../cliRunner";

it("should fail when trying to set role as mentionable", async () => {
  const command = Utils.buildCommandWithConfigPath("toSetRolePosition", "bot_case2.spec.ts");
  const [mockProcess, stdout] = await runCLI(command);
  expect(stdout).toMatchSnapshot();
  expect(mockProcess).toBeCalledWith(1);
});
