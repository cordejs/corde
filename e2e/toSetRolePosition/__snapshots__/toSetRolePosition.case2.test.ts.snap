// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should fail when trying to set role as mentionable 1`] = `
"
 FAIL   e2e\\\\toSetRolePosition\\\\__cordeTest__\\\\bot_case2.spec.ts   100ms
   x should fail when increase role position   100ms
    
    expected: role position to change to 5
    received: position didn't change

    at toSetRolePosition e2e\\\\toSetRolePosition\\\\__cordeTest__\\\\bot_case2.spec.ts:8:54
    at _internalTest (lib\\\\src\\\\closures\\\\test.js:29:13)
    at lib\\\\src\\\\closures\\\\test.js:52:65

Test Files: 1 failed. 1 total
Tests:      1 failed. 1 total
Time:       100ms
"
`;
